python/tests/voice/test_modality_stamps.py

```
@@ -0,0 +1,191 @@
+    """
+    from unittest.mock import patch as mock_patch
+    from scenario.voice.modality_resolver import resolve_modality
+
+    executor = _make_executor()
+
+    # Exercise resolve_modality with an explicit stt-bridge declaration.
+    # Patch litellm advisory so the test is deterministic (no network).
+    with mock_patch(
+        "scenario.voice.modality_resolver._litellm_advisory", return_value=False
+    ):
+        tier, warnings = resolve_modality(declaration="stt-bridge", model_id="openai/gpt-4o")
```

Ruff (0.15.15)

[warning] Unpacked variable `warnings` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)